format/Layout.md

```diff
@@ -41,7 +43,9 @@ Base requirements
   linearly in the nesting level
 * Capable of representing fully-materialized and decoded / decompressed Parquet
   data
-* All leaf nodes (primitive value arrays) use contiguous memory regions
+* All leaf nodes (primitive value arrays) use contiguous memory regions (TODO I'm not sure how to read this, 
```

Comment:
I think it's redundant and can be removed. I recall putting this in here strictly to reinforce the concept but it doesn't add meaning to the document